core/src/main/java/com/linecorp/armeria/common/MediaTypeNames.java

```diff
@@ -0,0 +1,155 @@
+/*
+ *  Copyright 2020 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License; charset=utf-8"; you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+public final class MediaTypeNames {
+
+    public static final String ANY_TYPE = "*/*";
```

Comment:
Missing Javadoc. Could use the string in verbatim, e.g.

```
/**
 * {@value #ANY_TYPE}.
 */
// or..
/**
 * {@code "*/*"}.
 */
```

Reply:
For all constants? What your intention is showing the value in javadoc?

I think first suggestion is better because it seems good to later when editing this code.

Reply:
When working with IDE, it will show the value of the constant in quick documentation, auto-completion hints, etc.